content/cloud/noops-sql-db.md

```diff
@@ -69,34 +69,63 @@ name = "c3"
 sqlite_databases = ["default"]
 ```
 
-The database will be created when the Spin application is deployed. The data persists Spin application invocations and updates. 
+The above configuration will last the lifespan of the application.
 
-```bash
-$ spin cloud deploy
-Deploying app ..
-Creating database inspirational-pig (default) for app "quickstart"
+### Preparing Fermyon Cloud Schema
+
+Once you have a NoOps SQLite Storage Spin application successfully built on localhost, prepare the appropriate Fermyon Cloud schema. You need to prepare Fermyon Cloud because your local Spin application's SQLite database is independent of your Cloud application's database. You can think of your local Spin application as a development and testing environment (with its own test data) and Fermyon Cloud as the production environment (with real user data). You can bootstrap your Cloud SQLite database schema (to match your local Spin application) in a couple of different ways.
+
+#### Source Code
```

Comment:
I would actually remove this section. It is possible to it this way, but I would certainly not consider it a "best practice". The "Spin Subcommand" section highlights the "best practice" way to migrate the database's schema.

Reply:
Thanks @rylev 
I have removed that paragraph and just retained the subcommand approach, as suggested.